src/magic_folder/magicfolderdb.py

```diff
@@ -215,19 +220,20 @@ def did_upload_version(self, relpath_u, version, last_uploaded_uri, last_downloa
                 action.add_success_fields(insert_or_update=u"update")
             self.connection.commit()
 
-    def store_local_snapshot(self, serialized_snapshot, name):
+    def store_local_snapshot(self, snapshot, name):
         """
```

Comment:
Can we just use `snapshot.name` in this method instead of passing name here (or, what should the caller pass for "name" if not `snapshot.name`?)

Reply:
pushed a commit.